### PR TITLE
feat: cleaner db

### PR DESCRIPTION
**Pull request by cnasc**
supersedes #3 

image

### CODE REVIEW

File: migrations/2023-03-17-132427_create_twabs/up.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+CREATE TABLE twabs (
```
Comment (cnasc) [inline migrations/2023-03-17-132427_create_twabs/up.sql:1]: will this only ever run on mainnet or do we want to track network here as well?

Comment (wilsoncusack) [reply migrations/2023-03-17-132427_create_twabs/up.sql:1]: good Q! I imagine we would have a dev env and db if we decide to track it there 🤔 . But could be simpler to combine

File: migrations/2023-03-17-132427_create_twabs/up.sql [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+    id SERIAL PRIMARY KEY,
+    currency_address CHAR(40) NOT NULL,
+    token_address CHAR(40) NOT NULL,
+    updated TIMESTAMP NOT NULL,
```
Comment (wilsoncusack) [inline migrations/2023-03-17-132427_create_twabs/up.sql]: created_at

**Comment by cnasc** [comment]
Unfortunately it looks like the migrations hasura wants are different from the migrations diesel creates for our test DB. Right now tracking both, not sure if there's a good way to unify them.

**Review by adamgobes — APPROVED** [review]

**Pull request merged**